README.md

```diff
@@ -1,52 +1,49 @@
-<img src="https://www.kubeflow.org/images/logo.svg" width="100">
+# Kubeflow
 
-[![Slack Status](https://img.shields.io/badge/slack-join_chat-white.svg?logo=slack&style=social)](https://www.kubeflow.org/docs/about/community/#kubeflow-slack-channels)
+[![Join Kubeflow Slack](https://img.shields.io/badge/slack-join_chat-white.svg?logo=slack&style=social)](https://www.kubeflow.org/docs/about/community/#kubeflow-slack-channels)
 [![CLOMonitor](https://img.shields.io/endpoint?url=https://clomonitor.io/api/projects/cncf/kubeflow/badge)](https://clomonitor.io/projects/cncf/kubeflow)
+[![OpenSSF Best Practices](https://www.bestpractices.dev/projects/2660/badge)](https://www.bestpractices.dev/projects/2660)
```

Comment:
Do we need this badge since `kubeflow/kubeflow` won't have any code base in the future ?

Reply:
We should probably have both, because google loves to index files like `CONTRIBUTING.md`.

Reply:
We can remove it then.

Reply:
Removed in https://github.com/kubeflow/kubeflow/pull/7642/commits/5513f05f5a67250aeb152b4154f1bb1ed523b727